src/metatrader/account_mt5.py

```diff
@@ -24,8 +24,8 @@ def get_account_profit(self) -> float:
     
     def get_positions(self) -> dict: # Note on 1.21.24 that MT5 returns positions as a named Tuple
         try:
-            pos = mt5.positions_get()
-            return { "positions": pos }
```

Comment:
how did this bug happen? All my previous commit versions had the 'return positions' statement. Going to approve

Reply:
It looks like it was introduced in [this PR](https://github.com/BensGitHub2022/MT5-Expert-Advisor/commit/9be6b2bd90a4f855586cd0e3f69e342786238d9a).

@bpsix, you'll want to adjust your APIs so that they turn the results into json in the web_service class since we are using the same data in the trading bots and want the original objects for them. Could you make a PR to update the code so that we are returning the original object for mt5 calls and then making them json inside of the relevant web_service method?